Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.mm

```diff
@@ -67,16 +67,29 @@
     return makeUnique<RemoteLayerTreeNode>(layerID, WTFMove(layer));
 }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+void RemoteLayerTreeNode::reconcileAfterScroll()
```

Comment:
Can we do this closer to the UIScrollView, like in `ScrollingTreeScrollingNodeDelegateIOS` ?